src/material/tooltip/tooltip.ts

```diff
@@ -147,6 +147,13 @@ export interface MatTooltipDefaultOptions {
 
   /** Disables the ability for the user to interact with the tooltip element. */
   disableTooltipInteractivity?: boolean;
+
+  /**
+   * Default classes to be applied to the tooltip. Supports the same syntax as `ngClass`. These
+   * default classes will not be applied if `tooltipClass` is defined directly on the tooltip
+   * element, as it will override the default.
+   */
+  tooltipClass?: string | string[] | Set<string> | {[key: string]: any};
```

Comment:
We should keep these to just `string | string[]` since they're static. Also we should remove the reference to `NgClass`.

Reply:
Nice one. Thanks for reviewing it :)